dc_make/maker.py

```diff
@@ -116,8 +118,48 @@ def getInputFile(self, era, model_params):
       self.input_files[file_name] = file
     return file_name, self.input_files[file_name]
 
+  def getRelevantBins(self, era, channel, category,signal_processes_histograms,unc_name=None, unc_scale=None, model_params=None):
+    relevant_bins = []
```

Comment:
```suggestion
    relevant_bins = set()
```
and at the end can be `return list(relevant_bins)` if you want to have a list